crates/genie-rec/src/unit.rs

```diff
@@ -277,34 +279,34 @@ impl StaticUnitAttributes {
             }
             members
         };
-        attrs.group_id = read_opt_u32(&mut input)?;
+        attrs.group_id = read_opt_u32(input)?;
         attrs.roo_already_called = input.read_u8()?;
         if input.read_u8()? != 0 {
-            attrs.sprite_list = Some(SpriteList::read_from(&mut input)?);
+            attrs.sprite_list = Some(SpriteList::read_from(input)?);
         }
         Ok(attrs)
     }
 
-    pub fn write_to(&self, mut output: impl Write, _version: f32) -> Result<()> {
-        output.write_u8(self.owner_id.into())?;
-        output.write_u16::<LE>(self.unit_type_id.into())?;
-        output.write_u16::<LE>(self.sprite_id.into())?;
-        todo!()
-    }
+    // fn write_to<W: Write>(&self, output: &mut W) -> Result<()> {
+    //     output.write_u8(self.owner_id.into())?;
+    //     output.write_u16::<LE>(self.unit_type_id.into())?;
+    //     output.write_u16::<LE>(self.sprite_id.into())?;
+    //     todo!()
+    // }
```

Comment:
Why was that commented out? Maybe small comment/todo/fixme? Or can it be deleted?

Reply:
I think because I wanted to have the new unimplemented thing, instead of the `todo()` but not have the code be Gone™